eventgrid.tf

```diff
@@ -0,0 +1,50 @@
+# Standard Topic
+resource "azurerm_eventgrid_topic" "function_app_eventgrid_topic" {
+  count               = var.eventgrid_topic_enabled == true ? 1 : 0
+  name                = "EG-${var.environment}-${var.application}"
+  location            = var.region
+  resource_group_name = data.azurerm_resource_group.main.name
+  tags                = module.tag_set.tags
+}
+
+# Standard Topic Subscription
+resource "azurerm_eventgrid_event_subscription" "function_app_eventgrid_topic" {
+  depends_on = [azurerm_eventgrid_topic.function_app_eventgrid_topic]
+  name       = "EGS-${var.environment}-${var.namespace}-${var.application}"
+  scope      = data.azurerm_resource_group.main.id
+  azure_function_endpoint {
+    function_id = azurerm_linux_function_app.linux_function[count.index]
```

Comment:
is it not applicable for window function app ?

Reply:
I didn't know we run windows, I saw it in there, and assumed that would not be what we were using. Updating this now